samples/SampleDestination/Startup.cs

```diff
@@ -5,34 +5,74 @@
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Logging;
 
 namespace SampleDestination
 {
     public class Startup
     {
+        private const string DefaultAllowedOrigin = "http://localhost:9001";
```

Comment:
I do believe there are issues using localhost for CORS, but I don't remember the details. @rynowak can probably comment on this. Given that you are using Kestrel. Could you use the machine name for this?

Reply:
If you listen on http://+:9001 you should be able to just to http://machine-name:9001 I believe

Reply:
I tried with the machine name and works fine, so I think it's better to use the machine name just in case.